src/pages/about/notices.md

```diff
@@ -23,7 +23,7 @@ This program may reference National Library of Medicine (NLM).
 <q>The duplication, sale, or redistribution of NLM data must conform to fair use guidelines and copyright law.</q>
 
 SNOMED Clinical Terms®
-Version: SNOMEDCT_US_2021_03_01 March 1, 2021
+Version: SNOMEDCT_US_2022_03_01 March 1, 2022
 
 This Program includes SNOMED Clinical Terms® (SNOMED CT®) which is used by permission of the International Health Terminology Standards Development Organisation (IHTSDO). All rights reserved. SNOMED CT®, was originally created by The College of American Pathologists. <q>SNOMED</q> and <q>SNOMED CT</q> are registered trademarks of the IHTSDO.
```

Comment:
Do we need the two sentences at line 28 which are duplicates of the last two sentences at line 30?